packages/block-editor/src/store/reducer.js

```diff
@@ -191,6 +191,24 @@ function withPersistentBlockChange( reducer ) {
 	return ( state, action ) => {
```

Comment:
Should we update the comment "All changes are considered persistent
 * except when updating the same block attribute as in the previous action" to include selection actions?